build.sbt

```diff
@@ -7,7 +7,8 @@ val circeVersion                = "0.14.3"
 val cirisVersion                = "2.4.0"
 val clueVersion                 = "0.23.1"
 val http4sVersion               = "0.23.16"
-val http4sJdkHttpClientVersion  = "0.5.0"
+val http4sBlazeVersion          = "0.23.12"
+val http4sJdkHttpClientVersion  = "0.7.0"
```

Comment:
I thought we'd normally use Ember client

Reply:
There were some issues with Ember a while back so I switched SSO to use Blaze. It may be ok now, I don't know.

Reply:
Yeah I tried and failed whereas blaze just worked.  I can use ember instead if it is important.  I may have some questions.

Reply:
Note this is just for the test case.

Reply:
Ember has had a lot of fixes lately. If there are more issues it would be good to fix those too.

Reply:
If it is for a test it doesn't really matter what we use

Reply:
> Yeah I tried and failed whereas blaze just worked

@swalker2m what exactly failed?

Reply:
@armanbilge  I'll try Ember again when I get a chance and if I can't figure it out I'll let you know.

Reply:
@armanbilge I [switched to Ember](https://github.com/gemini-hlsw/lucuma-itc/pull/320/commits/a51fd89e988e286067e4571142b84f4fef313742) and it works.  I had to change the default timeout though to keep the test from waiting for 30 seconds.  I would naively (and I am naive about this) expect a 0 second shutdown timeout to be fine, but that fails.  1 second seems to be fine but I set it to 2 for no good reason.